routes/controllers/imageUpload.js

```diff
@@ -0,0 +1,12 @@
+const multer = require('multer')
+const path = require('path')
+
+const storage  = multer.diskStorage({
+    destination: path.join(__dirname, '..', '..', 'public', 'uploads'),
+    filename: (res, file, cb) => {
+        const fileName = `${Date.now()}-${file.originalname}`
```

Comment:
Não é interessante manter o nome do arquivo aqui, isso pode usar usado para hacker alguma coisa, para essas imagens sempre são bom que definamos isso, também não ideal usar sequências numéricas, isso pode ser facilmente pego usando bruta força. 

vc poderia usar `uuid`
const { v4: uuidv4 } = require('uuid');
uuidv4(); // ⇨ '1b9d6bcd-bbfd-4b2d-9b5d-ab8dfbbd4bed'

https://www.npmjs.com/package/uuid